op-monitorism/faultproof_withdrawals/runbooks/RUNBOOK.md

```diff
@@ -0,0 +1,162 @@
+
+There are exceptions to the rule above. The event is still considered valid if:
+1. The [outputRoot provided](https://github.com/ethereum-optimism/optimism/blob/dd2b21ce786f4c1b722bda270348597182153c8e/packages/contracts-bedrock/src/L1/OptimismPortal2.sol#L314C15-L314C25) belongs to a FaultDisputeGame that is [blacklisted](https://github.com/ethereum-optimism/monitorism/blob/c0b2ecdf4404888e5ceccf6ad14e35c5e5c52664/op-monitorism/faultproof_withdrawals/validator/optimism_portal2_helper.go#L75), or to a FaultDisputeGame for which the game is in state [CHALLENGER_WIN](https://github.com/ethereum-optimism/monitorism/blob/c0b2ecdf4404888e5ceccf6ad14e35c5e5c52664/op-monitorism/faultproof_withdrawals/monitor.go#L326).
+2. The [outputRoot provided](https://github.com/ethereum-optimism/optimism/blob/dd2b21ce786f4c1b722bda270348597182153c8e/packages/contracts-bedrock/src/L1/OptimismPortal2.sol#L314C15-L314C25) belongs to a FaultDisputeGame that has the status [IN_PROGRESS](https://github.com/ethereum-optimism/monitorism/blob/c0b2ecdf4404888e5ceccf6ad14e35c5e5c52664/op-monitorism/faultproof_withdrawals/monitor.go#L336). We use the [faultproof_withdrawals_invalid_proposal_withdrawals_events_count](#faultproof_withdrawals_invalid_proposal_withdrawals_events_count) metric to track this event.
+
+#### Investigation
+To investigate this alert, we need to delineate the possible causes that generate it. This alert will trigger when a forgery is detected.
+
+The investigation has two parts:
+1. Investigation to triage if the event is a true positive.
+2. Investigation to act.
+
+TODO: Detail the following steps.
+
+### faultproof-withdrawal-forgery-detection-stalled
+
+| **Network** | **Severity Level** | **Impact**                                          | **Cause**                             | **Actions**                                                  |
+|-------------|--------------------|-----------------------------------------------------|---------------------------------------|--------------------------------------------------------------|
+| Mainnet     | SEV2               | - Security max reaction time is reduced<br>- May not be able to react properly to an attack | Daemon is not processing withdrawals  | - Understand the issue with the daemon<br>- If necessary, restart the service |
+| Sepolia     | SEV3               | - Security max reaction time is reduced<br>- May not be able to react properly to an attack | Daemon is not processing withdrawals  | - Understand the issue with the daemon<br>- If necessary, restart the service |
+
+##### Alert Description
+This alert monitors the number of withdrawal events that are considered normal in a chain. If the number of withdrawal events goes below a specified threshold, we trigger this alert.
+
+#### Investigation
+The investigation has two parts:
+1. Investigation to triage if the event is a true positive.
+2. Investigation to act.
+
+TODO: Detail the following steps.
```

Comment:
Same TODO here.

Reply:
Removed the sections that are incomplete. Will be added in a second moment